homework-3/src/main/java/com/epam/cashier/controller/service/repository/impl/ProductRepositoryImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.epam.cashier.controller.service.repository.impl;
+
+import com.epam.cashier.controller.service.model.Product;
+import com.epam.cashier.controller.service.repository.ProductRepository;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
```

Comment:
Please use @Repository annotation instead, actually it has no difference from @Component in high level, but it's more readable for reviewers

Reply:
Thank you, it was fixed